db/versions/0023.yml

```
@@ -0,0 +1,41 @@
+version: 23
+migrationScript: 0023-migration.sql
+downgradeScript: 0023-downgrade.sql
+methods:
+  add_github_accesstoken:
+    description: |-
+      Sets the access token for user_id_in to access_token_in.
+
+      If no access token is currently set for user_id_in, a new row is
+      inserted, otherwise the existing row is updated with a new access token
+      and etag, regardless of whether the access token has changed or not.
```

Just spotted my own mistake:

```suggestion
      inserted, otherwise the existing row's access token is updated to
      access_token_in, and a new etag is assigned regardless of whether the
      access token changed or not.
```